02-feature-engineering.Rmd

```diff
@@ -1247,12 +1247,13 @@ ggplot(pcs, aes(pc, percent_var)) +
   scale_y_continuous(expand = c(0, 0)) # move this to a book theme for plots
 ```
 
-A few things are of note here. First, it appears the majority of the variance is accounted for by our first few components, which is not altogether surprising given that we already new the first 10 components accounted for approximately 85% of the total variability. However, you might also wonder *why there are so many freaking components shown?! I thought we only asked for five!?* That is true. But the model actually estimates all the components, and then just pulls out whatever we ask, based on either the `threshold` or the `num_comp` arguments. For example, let's `bake` the five component recipe on the full training data.
+A few things are of note here. First, it appears the majority of the variance is accounted for by our first few components, which is not altogether surprising given that we already new the first 10 components accounted for approximately 85% of the total variability. However, you might also wonder *why there are so many freaking components shown?! I thought we only asked for five!?* That is true. But the model actually estimates all the components, and then just pulls out whatever we ask, based on either the `threshold` or the `num_comp` arguments. For example, let's `bake` the five component recipe on the full training data and print the resulting variables.
 
 ```{r }
 rec_5 %>% 
   prep() %>% 
-  bake(new_data = NULL)
+  bake(new_data = NULL) %>% 
+  names()
```

Comment:
I printed just the names here because I think it more clearly illustrates your point that only those five components are included in the resulting data. Otherwise the read has to read through the variable names in the collapsed tibble output, which isn't as obvious.